src/components/queryBuilder/Switch.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { InlineFormLabel, Switch as GrafanaSwitch, useTheme } from '@grafana/ui';
+import { styles } from 'styles';
+
+interface SwitchProps {
+  value: boolean;
+  onChange: (value: boolean) => void;
+  label: string;
+  tooltip: string;
+  inline?: boolean;
+}
+
+export const Switch = (props: SwitchProps) => {
```

Comment:
We use it only in a single place. Are you going to expand its usage instead of `GrafanaSwitch`?

Reply:
When I added this, I intended to use it for the "OTEL Logs" and "Live View" switches, the "OTEL Logs" switch was later moved to its own component that combined the version selector, leaving this one to be used with only "Live View"

It also had a strange style to it, so I corrected it by making this wrapper function, although I believe it's possible to fix using existing style classes.